definitions/npm/react-dnd_v2.x.x/flow_>=v0.23.x/react-dnd_v2.x.x.js

```diff
@@ -12,6 +12,8 @@ type DndOptions<P> = {
   arePropsEqual?: (props: P, otherProps: P) => boolean
 };
 
+type ElementOrNode = React$Element<any> | HTMLElement;
```

Comment:
I think `?HTMLElement` is more correct. When passing element, we should be able to `null` it out later.